.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,3 @@
+## Issue [INNO-XXXX](https://webgate.ec.europa.eu/CITnet/jira/browse/INNO-XXXX)
```

Comment:
Not sure about this one. When you have breaking changes in your PR, your PR message should start with `BREAKING CHANGE:`